s3fs/tests/test_s3fs.py

```diff
@@ -866,8 +914,9 @@ def test_errors(s3):
         f.close()
         f.read()
 
-    with pytest.raises(ValueError):
-        s3.mkdir("/")
+    # disabling this for now, mkdir("/") can be a non-op as creating bucket
+    # with pytest.raises(ValueError):
```

Comment:
Everything els looks good here, but I wonder about this one. Whilst I don't expect anyone to try to create "/", and we should no longer be accidentally doing it ourselves, it should still be an error, right? 
I am prepared to let it go as unimportant.

Reply:
You are right, i don't see any reason anyone would call this. Let me actually fix this.

Reply:
Can you please uncomment this part, so that your value error actually gets called?